generic/cloudbuild.yaml

```diff
@@ -1,12 +1,12 @@
 # gcloud builds submit --config=./generic/cloudbuild.yaml . --substitutions=_VERSION=0.8.0
 steps:
-- name: maven:3.6.2-jdk-8-slim
-  entrypoint: 'mvn'
-  args: ['package','-f','./generic/pom.xml']
-- name: gcr.io/cloud-builders/docker
-  args: ['build','-t', 'gcr.io/$PROJECT_ID/generic-pipeline:${_VERSION}', './generic/']
-- name: gcr.io/cloud-builders/docker
-  args: ['build', '-t', 'gcr.io/$PROJECT_ID/generic-pipeline:latest', './generic/']
+  - name: maven:3.6.2-jdk-8-slim
+    entrypoint: 'mvn'
+    args: ['package','-Dmaven.test.skip=true','-f','./generic/pom.xml']
```

Comment:
I'm not really happy for this, but we will have to fix later.

Reply:
Yes, sure. I'll add this as a task of high priority in my list.